metrotranscom/doorway
# feat: update footer with Doorway style and add links

# Pull Request Template

## Description

- Add Doorway footer by modifying the existing SiteFooter
- Add all new logos, exported from figma
- Add footer nav menu and external links
- Change content layout from two columns on desktop to two rows on mobile

One thing that I did differently from the specs is that the left edge of the content is not aligned between the bottom section of the footer and the top section. This happens because I used space-around to justify the top section since that works better for narrower desktop screens where otherwise the MTC logo would go into a new line.

Out of scope of this PR, will be added in a follow up
- Add footer to the listings page, which it was removed from.

## How Can This Be Tested/Reviewed?

- Bring up the public site
- Check the footer in the landing page and compare it to the specs
- Switch to mobile view and check the footer again

Desktop view: 
![image](https://github.com/metrotranscom/doorway/assets/67125998/44af943b-3345-4ab7-831b-088c459e5e5c)

Mobile view: 
![image](https://github.com/metrotranscom/doorway/assets/67125998/e6df84c5-feba-452b-8cc1-51278cfac984)
